canvas_modules/harness/src/client/constants/properties-documentation-constants.js

```diff
@@ -1099,6 +1099,122 @@ _defineConstant("TWISTY_PANEL_PROPS_INFO", {
 		}
 	}
 });
+_defineConstant("TEARSHEET_PANEL_PROPS_INFO", {
+	"title": "Panel Type: Tearsheet",
+	"parameterDef": {
+		"titleDefinition": {
+			"title": "Group: Tearsheet",
+			"editable": false
+		},
+		"current_parameters": {
+			"code": "Age >= 55"
+		},
+		"parameters": [
+			{
+				"id": "code",
+				"type": "string"
+			}
+		],
+		"uihints": {
+			"id": "code.test",
+			"label": {
+				"default": "Code Test"
+			},
+			"editor_size": "medium",
+			"description": {
+				"default": "Test code controls"
+			},
+			"parameter_info": [{
+				"parameter_ref": "code",
+				"language": "text/x-python",
+				"enable_maximize": true,
+				"data": {
+					"tearsheet_ref": "tearsheet1"
+				},
+				"label": {
+					"default": "Code"
+				},
+				"description": {
+					"default": "Enter Python code"
+				},
+				"text_before": {
+					"default": "Press ctrl-space to autocomplete"
+				},
+				"control": "code",
+				"class_name": "code-control-class"
+			}],
+			"group_info": [{
+				"id": "tearsheet-code",
+				"label": {
+					"default": "Expression with Maximize"
```

Comment:
I don't see `tearsheet-code` content on this screen. Only content from `tearsheet1` is displayed

![Screen Shot 2022-10-04 at 4 24 12 PM](https://user-images.githubusercontent.com/25124000/193948478-0a781a27-fffc-4cb0-a954-632fb147621e.png)